### PR TITLE
[OPPRO-346] Follow substrait's naming for bitwise functions

### PR DESCRIPTION
**Pull request by PHILO-HE**
This PR depends on https://github.com/substrait-io/substrait/pull/370.

test by CH[[238]]

### CODE REVIEW

File: gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala [hunk 1]
```diff
@@ -479,8 +479,8 @@ object ConverterUtils extends Logging {
   final val TAN = "tan"
   final val TANH = "tanh"
   final val BITWISE_NOT = "BitwiseNot"
-  final val BITWISE_AND = "BitwiseAnd"
-  final val BITWISE_OR = "BitwiseOr"
+  final val BITWISE_AND = "bitwise_and"
+  final val BITWISE_OR = "bitwise_or"
```
Comment (zzcclp) [inline gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala:483]: why ? it's better use consistent name for gluten, BitwiseNot, BitwiseAnd, why for Velox only ?

Comment (FelixYBW) [reply gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala:483]: What's the name in substrait? We should use the one in substrait, if substrait doesn't have it yet, we should submit a PR to add it, following substrait's function name pattern.

Comment (PHILO-HE) [reply gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala:483]: Got it. I just noted these functions are differently named in velox & CH. We have a pending PR to substrait. Let's follow the naming in substrait after that PR is merged. Thanks!
https://github.com/substrait-io/substrait/pull/370

**Comment by PHILO-HE** [comment]
@zzcclp, we have a patch merged to substrait (https://github.com/substrait-io/substrait/pull/370). Let's follow substrait's naming for bitwise functions. Could you please help do some fixing in CH? Thanks!

**Comment by zzcclp** [comment]
@loneylee please help to modify in CH, thanks.

**Comment by loneylee** [comment]
> @loneylee please help to modify in CH, thanks.

https://github.com/Kyligence/ClickHouse/pull/238

**Comment by zzcclp** [comment]
retest this please

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**